elfi/visualization/visualization.py

```diff
@@ -232,3 +233,86 @@ def plot_traces(result, selector=None, axes=None, **kwargs):
         axes[-1, ii].set_xlabel('Iterations in Chain {}'.format(ii))
 
     return axes
+
+
+def _diff(x, y):
+    return x - y
+
+
+def _abs_diff(x, y):
+    return abs(_diff(x, y))
+
+
+def compare(funs, *args, comparison='absdiff'):
+    """Plot a comparison of two functions.
```

Comment:
Typically when developing new ABC methods you are comparing it to some analytical solution.